docs/usage_and_configuration/black_docker_image.md

```diff
@@ -10,6 +10,10 @@ _Black_ images with the following tags are available:
 - `latest_release` - tag created when a new version of _Black_ is released.\
   ℹ Recommended for users who want to use released versions of _Black_. It maps to [the latest release](https://github.com/psf/black/releases/latest)
   of _Black_.
+- `latest_prerelease` - tag created when a new alpha (prerelease) version of _Black_ is
+  released.\
+  ℹ Recommended for users who want to preview or test alpha versions of _Black_. Note that
+  the most recent release may be newer than any prerelease.
```

Comment:
```suggestion
  the most recent release may be newer than any prerelease, because no prereleases
  are created before most releases.
```
I feel like this will make the statement easier to understand for most users.

Reply:
Lol I was wondering whether there'd be more changes to this. I agree that's clearer :+1: